index.html

```diff
@@ -506,6 +518,79 @@ <h3>
+          |this|'s [=randomized trigger rate=].</li>
+          <li>Set |confidence|'s {{PerformanceTimingConfidence/value}} as determined by the following algorithm:
+            <ol>
+              <li>Let |p| be |confidence|'s {{PerformanceTimingConfidence/randomizedTriggerRate}}.</li>
+              <li>Let |underlying| be [=this=]'s [=underlying confidence value=], a
+              {{PerformanceTimingConfidenceValue}}.</li>
+              <li>Let |r| be a real number drawn uniformly at random from the interval [0, 1).</li>
+              <li>If |r| &lt; |p|, return |underlying|.</li>
+              <li>Otherwise:
+                <ol>
+                  <li>Let |s| be an integer drawn uniformly at random from the set {0, 1}.</li>
+                  <li>If |s| equals 0, return {{PerformanceTimingConfidenceValue/high}}.</li>
+                  <li>Otherwise, return {{PerformanceTimingConfidenceValue/low}}.</li>
+                </ol>
+              </li>
+            </ol>
+          </li>
+          <li>Return |confidence|.</li>
+        </ol>
+        </p>
+        <div class="note">
+          <p>
+            These values should be set once, and not change for the lifetime of [=this=].
+          </p>
+        </div>
+        <div class="note">
+          <p>
+            This section is intended to help RUM providers and developers interpret
+            |confidence|. Since the [=randomized trigger rate=] can vary across records,
+            per-record weighting is needed to recover unbiased aggregates. The procedure
+            below illustrates how weighting based on {{PerformanceTimingConfidence/value}}
+            can be applied before computing summary statistics.
+          </p>
+          <p>
+            <ol>
+              <li>Let |p| be the record's {{PerformanceTimingConfidence/randomizedTriggerRate}}.</li>
+              <li>Let |c| be the record's {{PerformanceTimingConfidence/value}}.</li>
+              <li>
+                Assign a weight |w| based on |c|:
+                <ul>
+                  <li>If |c| is {{PerformanceTimingConfidenceValue/high}}, then |w| = <code>(1 - (|p| / 2)) / (1 - |p|)</code>.</li>
+                  <li>If |c| is {{PerformanceTimingConfidenceValue/low}}, then |w| = <code>(0 - (|p| / 2)) / (1 - |p|)</code>.</li>
+                </ul>
+              </li>
+              <li>Let |weighted_duration| = {{PerformanceEntry/duration}} * |w|.</li>
+            </ol>
+
+            To compute the mean for {{PerformanceTimingConfidenceValue/high}} records:
+            <ol>
+              <li>Let |total_high| be the sum of all |weighted_duration| values where |c| = high.</li>
+              <li>Let |sum_high_weights| be the sum of all |w| values where |c| = high.</li>
+              <li>Let |high_mean| = |total_high| / |sum_high_weights|.</li>
+            </ol>
+
+            To compute the mean for {{PerformanceTimingConfidenceValue/low}} records:
```

Comment:
Looks way better! It'd be neat to also outline how a certain percentile can be calculated

Reply:
Done. I also reworked the section on how to debias means to hopefully make that simpler.